soh/soh/Enhancements/sfx-editor/SfxEditor.cpp

```diff
@@ -208,7 +212,8 @@ void Draw_SfxTab(const std::string& tabId, const std::map<u16, std::tuple<std::s
             const auto& [name, sfxKey, seqType] = seqData;
             const std::string cvarKey = "gSfxEditor_" + sfxKey;
             if (seqType & type) {
-                if (((seqType & SEQ_BGM_CUSTOM) || seqType == SEQ_FANFARE) && defaultValue > MAX_AUTHENTIC_SEQID) {
+                // Only save authentic sequence CVars
+                if (((seqType & SEQ_BGM_CUSTOM) || seqType == SEQ_FANFARE) && defaultValue >= MAX_AUTHENTIC_SEQID) {
```

Comment:
just want to confirm the old `>` was a mistake and `>=` is what we want here

Reply:
Yes, there is a second instance below that has the >=

Reply:
gotcha, the naming is a little confusing, it's not clear that the authentic sequence with the highest id is 109 not 110 from this
https://github.com/HarbourMasters/Shipwright/blob/a6120811d5d270a322e2ec4e9fda01b1fee65b74/soh/include/z64audio.h#L29
the only place that's documented is here
https://github.com/HarbourMasters/Shipwright/blob/a6120811d5d270a322e2ec4e9fda01b1fee65b74/soh/src/code/audio_load.c#L1332